cloudstate/version.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding: utf-8 -*-
```

Comment:
This is safe?

Reply:
IMO yes. The default encoding for Python 3 is UTF-8, it was ASCII for Python 2. I [read](https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3) that one should use the header whenever one does not use the default encoding.

Reply:
@sleipnir wdyt?

Reply:
I like to sin for too much  :)